backend/RetryExamTestServer.js

```diff
@@ -179,7 +181,61 @@ define(["underscore", "moment-timezone", "PrairieRandom"], function(_, moment, P
                 qid: qid,
             });
         });
-        return tInstance;
+
+        // convert old Exam tInstances to RetryExams
+        if (tInstance.questionsByQID === undefined) {
+            tInstance.zones = [null]; // put everything into one zone
```

Comment:
By comparison with the code around L123, I think we just don't need to set `zones` or related properties for "old-style" `RetryExam`s.